mkdocs-project-dir/docs/howto.md

```diff
@@ -9,6 +14,7 @@ Logging in is most straightforward if you are inside the UCL firewall. If you ar
 ### Linux / Unix / Mac OS X
 
 Use the terminal and type the below command to secure shell (ssh) into the machine you wish to access. Replace `<your_UCL_user_id>` with your central UCL username, and `<system_name>` with the name of the machine you want to log in to, eg. `legion`, `grace`, `aristotle`. 
+<!--- myriad, kathleen, thomas, michael, young  --->
```

Comment:
I am using placeholders like this line,
an md-ignored-html-comment
because it does not render in localhost:8000 local pages; I need a non-rendered placeholder because I do not know what to put in this exact spot; I am effectively asking for advice "please provide the exact text to place here" because if I knew what to put, I would have put it and let it be contrasted / displayed in the diff here ...
Can you suggest an alternative, i.e. provide a line / comment / something that a) does not get rendered in localhost:8000 and is not an html comment?
My workflow as intended to be is to use those one-liners in the initial commit that starts a PR, and to make sure that by the second (that has to be the last as well)  commit, I will make sure that none survives; I do not work in an IDE or here in the browser to edit / commit / push, but in vim, and I make sure that no "<!--  " text pattern is left alive in the second commit while in :diffthis mode with the original ....
You may also have noticed that the first PR commit is lengthy in message and does not take the 'fixed / done / settled' wording ... it says 'spotted that error, suggested that change ...' After the second-which-is-last commit is done, with verbs changed to the 'fixed this / done that' wording in commit message, I see no obstacle for the PR handler to e.g., squash those commits into a new one with more concise message ...
e.g., https://gist.github.com/longtimeago/f7055aa4c3bba8a62197
https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git
or am I complicating things?

Reply:
The person accepting the PR should ideally just be able to click accept and merge, and not have to alter anything. (Ideally). Keep the amount of work they have to do to a minimum!

If they think one of your changes needs changes, they comment on it and request changes in the PR.